design-system-docs/frontend/styles/_foundations.scss

```diff
@@ -1,7 +1,16 @@
 // ============================================================================
-// Icons
+// Foundations
```

Comment:
Do you think it's worth me moving the contents of `sizing-border.scss` and `spacing-block.scss` here as they are part of the foundation? (as part of a separate PR)

Reply:
Yeah possibly. I originally did what you did an had separate files, but it started to feel simpler to just have a foundations.scss file so it's clear(er) these styles are one-offs.